app/views/transactions/_form.html.slim

```diff
@@ -13,8 +13,9 @@
       - else
         = f.association :category, as: :grouped_select, collection: current_organization.categories.grouped_by_type, group_method: :last, value_method: :id, prompt: 'Category', label: false
     .col-md-3.col-sm-6.col-xs-6
-      = f.input :customer_id, as: :select, include_blank: 'Customer',
-        label: false, collection: current_organization.customers
+      = f.input :customer_id, label: false,
+        input_html: { data: { url: customers_path } }
```

Comment:
@Gataniel Oj. I see..I think you should use only `name` field here. And on server side find client by name or create a new one.